src/amo/store.js

```diff
@@ -36,6 +39,10 @@ export default function createStore(initialState = {}) {
       search,
     }),
     initialState,
-    middleware(),
+    middleware({ sagaMiddleware }),
   );
+
+  store.runSaga = sagaMiddleware.run;
```

Comment:
Ten four. There are a few examples of this usage and it was quick and easy, but you're right. I'll go through and change it to export both items, it'll be a bit noisier in the diff.